integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/CarbonDescribeFormattedCommand.scala

```diff
@@ -105,6 +105,10 @@ private[sql] case class CarbonDescribeFormattedCommand(
     results ++= Seq(("Table Name: ", relation.tableMeta.carbonTableIdentifier.getTableName, ""))
     results ++= Seq(("CARBON Store Path: ", relation.tableMeta.storePath, ""))
     val carbonTable = relation.tableMeta.carbonTable
+    // Carbon table support table comment
+    val tableComment = carbonTable.getTableInfo.getFactTable.getTableProperties
+      .getOrDefault("comment", "")
```

Comment:
Move the `comment` as constant to CarbonCommonConstants

Reply:
Done.